velox/dwio/parquet/writer/Writer.h

```diff
@@ -141,6 +141,9 @@ class Writer : public dwio::common::Writer {
   void abort() override;
 
  private:
+  // Sets the memory reclaimers for all the memory pools used by this writer.
+  void setMemoryReclaimers(const std::shared_ptr<memory::MemoryPool>& pool);
```

Comment:
Do we need to pass pool to this class method?

Reply:
No, I removed it.